prime-dotnet-webapi/ViewModels/Business Events/SiteBusinessEventViewModel.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using Prime.Models;
+
+namespace Prime.ViewModels
+{
+    public class SiteBusinessEventViewModel
+    {
+        public int Id { get; set; }
+
+        public string AdminIDIR
+        {
+            get => Admin?.IDIR;
+        }
+
+        public Admin Admin { get; set; }
```

Comment:
Do we need the `Admin` object here? I assume that we only need the `Admin`'s IDIR, meaning you could do a mapping just to get the IDIR in the same way you did the mapping to get the Signing Authority's name